newsletters/2025-02.md

```diff
@@ -0,0 +1,63 @@
+---
+tags: obf
+---
+
+_Having trouble viewing this email or looking for a permalink? [View this newsletter in your browser](https://github.com/OBF/newsletter/blob/master/newsletters/2025-02.md)._
+
+_This email was sent to the [bosc-announce](https://groups.google.com/g/bosc-announce) and [open-bio-l](http://mailman.open-bio.org/mailman/listinfo/open-bio-l/) mailing lists._
+
+# OBF Newsletter February 2025
+
+TL;DR: Updates from BOSC 2025, changes to the OBF fellowship and community news!
+
+In longer form: 
+
+## OBF News
+
+### BOSC news
+
+#### BOSC 2025
+
+This year, BOSC will take place July 21-22, 2025 in Liverpool, UK (as part of ISMB/ECCB 2025) and will cover all aspects of open source bioinformatics and open science.
```

Comment:
```suggestion
Our annual conference, the [Bioinformatics Open Source Conference](https://www.open-bio.org/events/bosc-2025/) (BOSC), will take place July 21-22, 2025 in Liverpool, UK as part of ISMB/ECCB 2025. This will be the 26th BOSC; as always, it will cover all aspects of open source bioinformatics and open science.
```